app/src/main/java/ch/epfl/sdp/contamination/Carrier.java

```diff
@@ -5,16 +5,12 @@
     /**
      * Represents the possible stages of infection:
      * HEALTHY:         Healthy carrier, not infected, not suspected to be ill, not immune
-     * HEALTHY_CARRIER: Know to have been infected, not yet ill but contagious
      * INFECTED:        Definitely ill and contagious
-     * IMMUNE:          Healthy carrier, either healed or already immune for other reasons
      * UNKNOWN:         Ill/healthy with some probability (no clues, bad luck :( )
      */
     enum InfectionStatus {
         HEALTHY,
-        HEALTHY_CARRIER,
         INFECTED,
-        IMMUNE,
         UNKNOWN
```

Comment:
Could you explain why you removed the HEALTHY_CARRIER and IMMUNE status? Why aren't they relevant anymore?

Reply:
Ok we discussed that point with Matteo, now you are never "immune" at 100 %, but you are immune according to the equation I mentioned in the top of the PR. 
For HEALTHY_CARRIER: this possibility was also already captured by my last PR about infectionAlerts. Remember : HEALTHY_CARRIER is about carrying the disease without notifying it, which is kind of the same of my last PR